build.sbt

```diff
@@ -33,10 +33,25 @@ lazy val troyMacro = project
     Library.cassandraUnit % Test
   ))
   .dependsOn(troyDriver, troySchema)
-  .settings((Defaults.coreDefaultSettings ++ Seq(
+  .settings(Defaults.coreDefaultSettings ++ Seq(
     unmanagedClasspath in Test ++= (unmanagedResources in Test).value,
     parallelExecution in Test := false
-  )) : _*)
+  ): _*)
+
+lazy val troyMeta = project
+  .in(file("troy-meta"))
+  .settings(libraryDependencies ++= Vector(
```

Comment:
It is a convention from [sbt-fresh](https://github.com/sbt/sbt-fresh/blob/master/build.sbt)